src/app/kbv/steps.js

```diff
@@ -26,6 +27,19 @@ module.exports = {
     controller: singleAmountQuestionController,
     next: "load-question",
   },
+  "/prove-identity-another-way": {
+    resetJourney: true,
+    entryPoint: true,
+    controller: proveIdentityAnotherWayController,
+    prereqs: ["/kbv/load-question"],
+    next: [
+      {
+        value: "continue",
+        next: "single-amount-question",
```

Comment:
Shouldn't this be `load-question` (which will work out which question to go to) ?

Reply:
In fact, it'll be more complicated than this as it will need to work out whether to go to a question (if came from question page) or the start page (if came from start page) - suggest passing this through a query param